exercicios/exercicio5.js

```diff
@@ -1 +1,8 @@
-const notas = [5, 4, 3, 0, 10]
+
+var notas = [5, 4, 3, 0, 10]
+var total = 0
+    for (let i = 0; i < notas.lenght; i++){
```

Comment:
Boa! Um ponto de atenção aqui é a indentação. Essa linha tem uma indentação de 4 espaços, mas deveria ser 0 porque esse `for` não está dentro de nenhuma outra estrutura. A mesma coisa pras outras linhas abaixo!

Além disso, o que acha de tentar resolver esse exercício agora utilizando `reduce`? (: